<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
  </property>
  <widget class="QLineEdit" name="txtPath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Source path</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>165</width>
     <height>21</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Change picture every</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinUpdateInterval">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>minutes</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
